development/kops/create_values_yaml.sh

```diff
@@ -85,6 +85,8 @@ kubernetesVersion: ${ARTIFACT_URL}/kubernetes-${RELEASE_BRANCH}/releases/${RELEA
 clusterName: $KOPS_CLUSTER_NAME
 configBase: $KOPS_STATE_STORE/$KOPS_CLUSTER_NAME
 awsRegion: $AWS_DEFAULT_REGION
+controlPlaneInstanceProfileArn: ''
```

Comment:
Wondering if this will lead to duplicate fields in values.yaml if the profile ARN's are set as environment variables 🤔

Reply:
Good point, I forgot that stuff was down there

Reply:
Fixed